app/v1/connectors/DeleteUkDividendsIncomeAnnualSummaryConnector.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import api.connectors.{BaseDownstreamConnector, DownstreamOutcome}
+import api.connectors.DownstreamUri.DesUri
+import config.AppConfig
+import javax.inject.{Inject, Singleton}
+import play.api.libs.json.Json
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
+import v1.models.request.deleteUkDividendsIncomeAnnualSummary.DeleteUkDividendsIncomeAnnualSummaryRequest
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+case class DeleteUkDividendsIncomeAnnualSummaryConnector @Inject() (http: HttpClient, appConfig: AppConfig) extends BaseDownstreamConnector {
```

Comment:
No need for this to be a case class:
```suggestion
class DeleteUkDividendsIncomeAnnualSummaryConnector @Inject() (http: HttpClient, appConfig: AppConfig) extends BaseDownstreamConnector {
```